sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayoutController.java

```diff
@@ -108,6 +110,8 @@
   private List<Pair<String, WritableMap>> queuedEvents;
   private boolean isReactMounted;
   private boolean isTargetTV;
+  private RecyclerView rootRecyclerView = null;
+  private View.OnLayoutChangeListener recyclerLayoutChangeListener = null;
```

Comment:
`recyclerLayoutChangeListener ` has been never used.

Reply:
yes, forgot to remove it after some testing , done